docs/technical_reference/unit_models/zero_order_unit_models/mabr_zo.rst

```diff
@@ -16,6 +16,26 @@ Costing Method
 Costing is calculated using the cost_mabr method in the zero-order costing package.
 See documentation for the :ref:`zero-order costing package<zero_order_costing>`.
 
+Additional Variables
+--------------------
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Units"
+
+   "Nitrogen removal rate per day", "nitrogen_removal_rate", "g/d/m**2"
+   "Sizing variable for effective reactor area", "reactor_area", "m**2"
```

Comment:
This might just be GitHub, but this appeared in bold as I was reviewing it.

Reply:
I originally thought this was an issue too when looking at this through Github. However, when I checked via read the docs, the rendering is fine.